p4-fusion/labels_conversion.cc

```diff
@@ -0,0 +1,212 @@
+#include <string>
+
+#include "p4_api.h"
+#include "git2/commit.h"
+#include "git2/refs.h"
+#include "git2/repository.h"
+#include "git2/types.h"
+
+std::string sanitizeLabelName(std::string label)
```

Comment:
could we add a docstring to this method that outlines what this function does, without needing to read the code?